site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/IdpOrgSyncPage.tsx

```diff
@@ -0,0 +1,111 @@
+import { getErrorMessage } from "api/errors";
+import {
+	organizationIdpSyncSettings,
+	patchOrganizationSyncSettings,
+} from "api/queries/idpsync";
+import type { OrganizationSyncSettings } from "api/typesGenerated";
+import { ChooseOne, Cond } from "components/Conditionals/ChooseOne";
+import { displayError } from "components/GlobalSnackbar/utils";
+import { displaySuccess } from "components/GlobalSnackbar/utils";
+import { Loader } from "components/Loader/Loader";
+import { Paywall } from "components/Paywall/Paywall";
+import { SquareArrowOutUpRight } from "lucide-react";
+import { useDashboard } from "modules/dashboard/useDashboard";
+import { useFeatureVisibility } from "modules/dashboard/useFeatureVisibility";
+import { type FC, useEffect } from "react";
+import { Helmet } from "react-helmet-async";
+import { useMutation, useQuery, useQueryClient } from "react-query";
+import { docs } from "utils/docs";
+import { pageTitle } from "utils/page";
+import { ExportPolicyButton } from "./ExportPolicyButton";
+import IdpOrgSyncPageView from "./IdpOrgSyncPageView";
+
+export const IdpOrgSyncPage: FC = () => {
+	const queryClient = useQueryClient();
+	// IdP sync does not have its own entitlement and is based on templace_rbac
+	const { template_rbac: isIdpSyncEnabled } = useFeatureVisibility();
+	const { organizations } = useDashboard();
+	const {
+		data: orgSyncSettingsData,
+		isLoading,
+		error,
+	} = useQuery(organizationIdpSyncSettings());
+
+	const patchOrganizationSyncSettingsMutation = useMutation(
+		patchOrganizationSyncSettings(queryClient),
+	);
+
+	useEffect(() => {
+		if (patchOrganizationSyncSettingsMutation.error) {
+			displayError(
+				getErrorMessage(
+					patchOrganizationSyncSettingsMutation.error,
+					"Error updating organization idp sync settings.",
+				),
+			);
+		}
+	}, [patchOrganizationSyncSettingsMutation.error]);
+
+	if (isLoading) {
+		return <Loader />;
+	}
+
+	return (
+		<>
+			<Helmet>
+				<title>{pageTitle("Organization IdP Sync")}</title>
+			</Helmet>
+
+			<div className="flex flex-col gap-12">
+				<header className="flex flex-row items-baseline justify-between">
+					<div className="flex flex-col gap-2">
+						<h1 className="text-3xl m-0">Organization IdP Sync</h1>
+						<p className="flex flex-row gap-1 text-sm text-content-secondary font-medium m-0">
```

Comment:
in the designs, most fonts are 500 but on this page it only needs to be specified here

Reply:
The reason why I was asking was because all the other pages use font weight 400 (Roman weight). I think it's outside the scope of the PR, but we'll need to make another PR to update those, too

Reply:
yes there is an issue already created to handle global style changes